flake.nix

```diff
@@ -1,9 +1,6 @@
 {
   inputs = {
-    flake-compat = {
-      url = "github:edolstra/flake-compat";
-      flake = false;
-    };
+    flake-compat.url = "https://flakehub.com/f/edolstra/flake-compat/1.tar.gz";
```

Comment:
they recommend using this over the github url. i don't think there is much of a difference in this case since the repo isn't really receiving updates.